### DISCUSSION TITLE
FreeTube on KODI

### DISCUSSION THREAD

**Discussion by EtherGhost** [FreeTubeApp/FreeTube#1950] [Q&A]
Is it possible to use FreeTube on Kodi?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No. An app for Kodi would have to be made from scratch to support.

**Reply by EtherGhost** [reply]
Ok, thanks

**Comment by jkirk** [comment]
I just found this: https://github.com/georgehuble/plugin.video.freetube

The main code is here: https://github.com/georgehuble/plugin.video.freetube/blob/master/resources/lib/service.py

I haven't tested the Kodi plugin yet, but I'd love to hear what the FreeTube devs think about the approach. Thx!

**Reply by absidue** [reply]
@jkirk The "FreeTube" name in that project is completely misleading, it has nothing to do with FreeTube. They seem to have taken inspiration from FreeTube but that's the only link, it's a completely separate project, written in a completely different programming language, functions differently and stores it's data in a completely different way.

**Reply by jkirk** [reply]
@absidue Thx! Thank you for your feedback, that's what I wanted to know.

The naming could be unintentionally (or not), but maybe we should reach out to @georgehuble and kindly ask to rename the plugin to something else to avoid confusion.

**Reply by Ein-Tim** [reply]
This was done via https://github.com/georgehuble/plugin.video.freetube/issues/2
